sdk/identity/azure-identity/src/main/java/com/azure/identity/ChainedTokenCredential.java

```diff
@@ -63,4 +62,14 @@
                        return Mono.error(last);
                    }));
     }
+
+
+    /**
+     * Get the read-only list of credentials sequentially used to attempt authentication.
+     *
+     * @return The list of {@link TokenCredential}.
+     */
+    public List<TokenCredential> getCredentials() {
+        return credentials;
```

Comment:
The description here says that the list is read-only is this the case? Since it's returning the underlying instance of `credentials` as a `List<TokenCredential>`? We want to make sure we don't allow people to mutate the state of the already created credential.

Reply:
Discussed this offline, and since we're using `Colletions.unmodifiableList` to create this instance we can return it without worrying about users modifying it.